Project/Assets/ML-Agents/Examples/Bouncer/Scripts/BouncerAgent.cs

```diff
@@ -42,9 +42,17 @@ public override void OnActionReceived(ActionBuffers actionBuffers)
             continuousActions[i] = Mathf.Clamp(continuousActions[i], -1f, 1f);
         }
         var x = continuousActions[0];
-        var y = ScaleAction(continuousActions[1], 0, 1);
+        var y = 0f;
+        if (continuousActions[1] > 0 && m_JumpCooldown <= 0f)
+        {
+            m_JumpLeft -= 1;
+            m_JumpCooldown = 0.1f;
```

Comment:
I think this will cause issues for training since RequestDecision would be called a lot of times in succession as long as the action 1 is 0. This could confuse training when using behavioral cloning as a lot of actions from the human player would be 0 actions (and the trained agent would be unable to have its action be exactly 0). 

Also, the Bouncer Agent was made to bounce, not sure making it stick to the platform when a human is playing is desirable.

Reply:
Maybe my solution is not the best, but I still think there is some issue as the m_JumLeft is decreasing even when the action to jump is 0. If the Bouncer needs to jump all the time, we don't need this action, it could be 1 all the time, or just remove it from the model

Reply:
What if we replace line 45 with 
```
var y = ScaleAction(continuousActions[1], 0.1f, 1);
```
instead of
```
var y = ScaleAction(continuousActions[1], 0, 1);
```

This way the vertical action is never 0?